src/AccessibilityInsights/MainWindowHelpers/AutoUpdate.cs

```diff
@@ -193,32 +192,14 @@ private static string GetTimeSpanTelemetryString(TimeSpan? timeSpan)
             return timeSpan.HasValue ? timeSpan.Value.ToString() : "unknown";
         }
 
-        private static string GetMessageForOptionalUpdateFailure(UpdateResult updateResult)
+        private static string GetMessageForOptionalUpdateFailure()
         {
-            switch (updateResult)
-            {
-                case UpdateResult.DownloadFailed:
-                    return Messages.OptionalUpdateFailedOnDownload;
-                case UpdateResult.VerificationFailed:
-                    return Messages.OptionalUpdateFailedVerification;
-                case UpdateResult.Unknown:
-                default:
-                    return Messages.OptionalUpdateFailedUnknown;
-            } // switch
+            return Messages.OptionalUpdateFailedUnknown;
```

Comment:
The old code had different messages for mandatory and optional upgrades. The new code only tells if the version switcher failed. Version switcher doesn't yet know about the concept of a mandatory upgrade